spec/features/ckeditor_spec.rb

```diff
@@ -30,4 +30,14 @@
 
     expect(page).to have_css "img[src$='clippy.jpg']"
   end
+
+  scenario "cannot upload attachments through link tab", :js do
+    login_as(create(:administrator).user)
+    visit new_admin_site_customization_page_path
+
+    find(".cke_button__link").click
+
+    expect(page).not_to have_link "Upload"
```

Comment:
This test passes even without the changes in this pull request :thinking:.

I think we need to add an extra expectation before this line. Otherwise this line might be executed before CKEditor's dialog appears, and pass because at that time thi "Upload" link is still not present.

How about:

```ruby
    expect(page).to have_css(".cke_dialog")
    expect(page).not_to have_link "Upload"
    expect(page).not_to have_link "Browse Server"
```

Reply:
Hi @javierm you are right, i swear yesterday i saw this test failing in my machine! Maybe i forgot to save a file before running the spec. Now i added the first expectation you suggested! Thanks a lot